plsc/entry.py

```diff
@@ -293,8 +308,12 @@ def _get_optimizer(self):
     
         if self.loss_type in ["dist_softmax", "dist_arcface"]:
             self.optimizer = DistributedClassificationOptimizer(
-                self.optimizer, global_batch_size)
-
+                self.optimizer, global_batch_size, use_fp16=self.use_fp16,
+                loss_type=self.loss_type,
+                fp16_user_dict=self.fp16_user_dict)
+        elif self.use_fp16:
```

Comment:
how to use both dist_xxx and use_fp16?

Reply:
if use dist_xxx, DistributedClassificationOptimizer will be used no matter when use_fp16 equals True or not.
if use other loss type, decorate function will be called if use_fp16 equals True.